docs/develop/node/archival/run-archival-node-without-nearup.md

```diff
@@ -0,0 +1,277 @@
+---
+id: run-archival-node-without-nearup
+title: Run an Archival Node without nearup
+sidebar_label: Run a Node without nearup
+description: How to run an Archival Node without nearup
+---
+
+<blockquote class="info">
+<strong>Heads up</strong><br><br>
+
+Running an archival node is very similar to running a [validator node](/docs/develop/node/validator/running-a-node) as both types of node use the same `nearcore` release. The main difference for running an archival node is a modification to the `config.json` by changing `archive` to `true`. See below for more details.
+
+</blockquote>
+
+
+## Prerequisites
+
+- [Rust](https://www.rust-lang.org/). If not already installed, please [follow these instructions](https://docs.near.org/docs/tutorials/contracts/intro-to-rust#3-step-rust-installation).
+- [Git](https://git-scm.com/)
+- Installed developer tools:
+  - MacOS
+    ```bash
+    $ brew install cmake protobuf clang llvm
+    ```
+  - Linux
+    ```bash
+    $ apt update
+    $ apt install -y git binutils-dev libcurl4-openssl-dev zlib1g-dev libdw-dev libiberty-dev cmake gcc g++ python docker.io protobuf-compiler libssl-dev pkg-config clang llvm
+    ```
+---
+
+### Choosing your `nearcore` version
+
+When building your NEAR node you will have two branch options to choose from depending on your desired use:
+
+- `master` : _(**Experimental**)_
+  - Use this if you want to play around with the latest code and experiment. This branch is not guaranteed to be in a fully working state and there is absolutely no guarantee it will be compatible with the current state of *mainnet* or *testnet*.
+- [`Latest release branch`](https://github.com/near/nearcore/releases) : _(**Stable**)_
+  - Use this if you want to run a NEAR node for *mainnet* or *tesnet*. This version is used by validators and other nodes and is fully compatible with the current state of *mainnet* or *testnet*.
```

Comment:
This is not completely true. When we make a release, we first release a rc version for testnet and then later make that release stable. So for testnet, running the latest stable version does not always work. We should change it to running the latest rc version